src/styles/block/cards.scss

```diff
@@ -0,0 +1,108 @@
+.main {
+  box-sizing: border-box;
+  margin: 0 auto;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: 48px ;
+  max-width: 1024px;
+
+  .card {
+    display: flex;
+    flex-direction: column;
+
+    width: 198px;
+    border: 1px solid $borderInCard;
+    border-radius: 5px;
+    transition: $transition-all;
+
+    &:hover {
+      transform: scale(1.2);
+    }
+
+    &__image {
+      margin: 33px 19px 0;
+    }
+
+    &__photo {
+      display: block;
+      max-width: 100%;
+      height: auto;
+    }
+
+    &__container {
+      margin: 40px 16px 16px;
+    }
+
+    &__title {
+      padding-bottom: 4px;
+
+      &-h2 {
+        font-weight: 500;
+        font-size: $fsInTitle;
+        line-height: 18px;
+        margin: 0 ;
+
+        &:hover {
+          color: $textNavLinkHover;
+        }
+      }
+    }
+
+    &__subtitle {
+      font-size: 10px;
+      line-height: 14px;
+      color: $color-grey;
+      padding-bottom: 16px;
+    }
+
+    &__reviews {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      color: $color-black;
+      padding-bottom: 24px;
+    }
+
+    &__price {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      padding-bottom: 16px;
+
+      &-left {
+        font-size: $fsInTitle;
+        line-height: 18px;
+        color: $color-grey;
+      }
+
+      &-right {
+        font-size: 16px;
+        line-height: 18px;
+        font-weight: 700;
+      }
+    }
+
+    &__link-button {
+      display: block;
+      text-align: center;
+      margin: 0 auto;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)